src/Security/InteractiveSecurityHelper.php

```diff
@@ -75,4 +77,22 @@ public function guessEntryPoint(SymfonyStyle $io, array $securityData, string $a
             current($authenticators)
         );
     }
+
+    public static function guessUserClass(SymfonyStyle $io, array $securityData): string
+    {
+        if (1 === \count($securityData['security']['providers']) && isset(current($securityData['security']['providers'])['entity'])) {
+            $entityProvider = current($securityData['security']['providers']);
+            $userClass = $entityProvider['entity']['class'];
```

Comment:
Just `return $entityProvider['entity']['class'];`. Then the next part doesn't need to be in the `else`